css/style.css

```diff
@@ -0,0 +1,204 @@
+/*  Add your styling here */
+
+header{
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+}
+.sec-logo{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    background-color: rgb(247, 218, 248);
+
+}
+.sec-menu{
+    text-align: end;
+    margin: 40px;
+    background-color:#ffafcc;
+}
+.sec-menu ul li{
+    display: inline-block;
+    list-style-type: none;
+    font-size: 0.9rem;
+    padding: 5px 15px;
+    margin-right: 10px;
+    background-color: #a2d2ff;
+}
+.sec-menu a{
+    text-decoration: none;
+    color: white;
+}
+.sec-menu a:hover{
+    color: black;
+}
+.div-logo{
```

Comment:
div-logo is not a great naming, try to have descriptive names, like here you can call it cake-logo or bakery-logo